README.md

```diff
@@ -103,7 +103,25 @@ docker run -p 8080:8080 philipssoftware/bom-base:latest
 ## How to test the software
 
 Unit tests are executed by the Maven `mvn clean test` command.
-
+Test requires scancode-toolkit to be installed.
+
+find how to install and configure based on OS at https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#source-code-install 
+
+On Linux/Mac:
+1. Open a terminal
```

Comment:
Developers would know this step (and the one below), so I don't think it is necessary to mention it here.